src/Sarif.Viewer.VisualStudio.ResultSources.GitHubAdvancedSecurity.Core/Services/GitHubSourceService.cs

```diff
@@ -58,6 +58,7 @@ public partial class GitHubSourceService : IResultSourceService, IGitHubSourceSe
         private const string GetAnalysesEndpoint = "analyses";
         private const string DismissAlertEndpointFormat = "alerts/{0}";
         private const string GitLocalRefFileBaseRelativePath = @".git\refs\remotes\origin";
+        private const string OptionStateKey = "GitHubAdvancedSecurity";
```

Comment:
>GitHubAdvancedSecurity [](http://example.com/codeflow?start=47&length=22)

nit: define the const in a common place, SarifViewerOptions and this class can access.

Reply:
Agreed, but I'm going to address this kind of thing in the restructuring. Too much of a pain currently.